scripts/commitlint/index.js

```diff
@@ -0,0 +1,51 @@
+const { Octokit } = require('@octokit/rest')
+const lint = require('@commitlint/lint')
+
+const { rules } = require('../../commitlint.config')
+
+function getPrNumber() {
+  const prUrl = process.argv[2]
+  const prUrlParts = prUrl.split('/')
+
+  return prUrlParts[prUrlParts.length - 1]
+}
+
+function getCommitsByPr(number) {
+  const github = new Octokit()
+  return github.pulls.listCommits({
+    owner: 'Royal-Navy',
+    repo: 'standards-toolkit',
+    pull_number: number,
+  })
+}
+
+function lintCommits(commits) {
+  const promises = commits.data.map(commit => {
+    return lint(commit.commit.message, rules)
+  })
+
+  return Promise.all(promises)
+}
+
+const runCommitLint = async () => {
+  try {
+    const prNumber = getPrNumber()
+    const prCommits = await getCommitsByPr(prNumber)
+    const lintResults = await lintCommits(prCommits)
+
+    const resultsWithErrors = lintResults.filter(result => !result.valid)
+    resultsWithErrors.forEach(resultsWithError => {
```

Comment:
The naming of the lambda param here (`resultsWithErrors`) confused me a bit when I was scanning.

Maybe we could destructure the error bag out of it?
```suggestion
    resultsWithErrors.forEach(({ errors }) => {
```